platforms/core-configuration/model-core/src/main/java/org/gradle/api/internal/provider/CollectionPropertyExtensions.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.internal.provider;
+
+import org.gradle.api.provider.HasMultipleValues;
+import org.gradle.api.provider.Provider;
+
+@SuppressWarnings("unused") // registered as Groovy extension in ExtensionModule
+public class CollectionPropertyExtensions {
+
+    /**
+     * Adds an element to the property value
+     *
+     * <p>Extension method to support the left shift operator in Groovy.</p>
```

Comment:
💅 Maybe using the actual operator will be clearer.

```suggestion
     * <p>Extension method to support the {@code <<} operator in Groovy.</p>
```